controllers/articles.controller.js

```diff
@@ -12,12 +12,18 @@ const getArticles = (request, response) => {
 
 const getArticleByIdController = (request, response) => {
     const { article_id } = request.params;
-    getArticleById(article_id)
-    .then(({ rows, rowCount }) => {
-        if(rowCount === 0) {
-            response.status(404).send({ msg: 'Article not found'})
+    if (Number.isNaN(parseInt(article_id, 10))) {
+        response.status(400).send({ msg: 'Bad request' });
```

Comment:
We should be looking to handle these with custom error handling middleware. I recommend looking at the Northcoders notes for this